features/package-install.feature

```diff
@@ -160,6 +160,16 @@ Feature: Install WP-CLI packages
       Removing require statement from {PACKAGE_PATH}composer.json
       Removing repository details from {PACKAGE_PATH}composer.json
       """
+    And the {PACKAGE_PATH}composer.json file should not contain:
```

Comment:
The way you're testing this won't work reliably, because the content you're checking is too specific.

You're not only checking whether the repository entry is in there, but also that it has the require statement before it (which is not possible in the way you're testing) and that all indentation and formatting is exactly as provided.

I suspect that you will never be able to have this test item fail, even if you remove the code you've built.

You should test for a string of text that does not make any other assumptions instead, like:

 ```
   And the {PACKAGE_PATH}composer.json file should not contain:
   """
   "wp-cli/google-sitemap-generator-cli": "dev-master"
   """
   And the {PACKAGE_PATH}composer.json file should not contain:
   """
   "url": "git@github.com:wp-cli/google-sitemap-generator-cli.git"
   """
```

Reply:
Makes sense @schlessera, I updated the test